controllers/s3_store_accessors.go

```diff
@@ -41,11 +36,7 @@ func s3StoreAccessorsGet(
 
 		s3StoreAccessors = append(s3StoreAccessors, s3StoreAccessor{
 			objectStorer,
-			s3ProfileName,
-			s3StoreProfile.S3CompatibleEndpoint,
-			s3StoreProfile.S3Bucket,
-			s3StoreProfile.S3Region,
-			s3StoreProfile.VeleroNamespaceSecretKeyRef,
+			s3StoreProfile,
```

Comment:
It's not obvious to me why this change is needed in this PR, but it's good refactoring. I believe the requests.go could use similar treatment so it's not required to pass in seven or so values for creating a backup/restore request, but that is optional and can/should be deferred.

Reply:
One more field that had to be duplicated was enough for me to fix it.  The kubeobjects interface is intended to be a separate "library" and not have any dependencies on Ramen including its `S3StoreProfile`.